api.docs/includes/_flows.md

```diff
@@ -359,6 +359,66 @@ Type | Description
 --------- | ---- | ------- | -----------
 <a href="#flowresult">FlowResult</a> | An error object or empty
 
+## Done Edit a Flow
+
+```graphql
+mutation doneEditFlow($id: ID!) {
```

Comment:
This looks a bit confusing. Can we rename this API endpoint something like **publish_flow**

Reply:
Yes, that makes sense.